core/consensus/babe/impl/babe_impl.cpp

```diff
@@ -264,20 +264,18 @@ namespace kagome::consensus::babe {
   void BabeImpl::processSlot() {
     BOOST_ASSERT(keypair_ != nullptr);
 
-    if (not slots_leadership_.has_value()) {
+    if (not(lottery_->epoch() == current_epoch_)) {
```

Comment:
Why not `lottery_->epoch() != current_epoch_`

Reply:
operation not defined